packit_service/config.py

```diff
@@ -272,17 +272,26 @@ def get_project_to_sync(self, dg_repo_name, dg_branch) -> Optional[ProjectToSync
 class PackageConfigGetter:
     @staticmethod
     def create_issue_if_needed(
-        project: GitProject, title: str, message: str
+        project: GitProject,
+        title: str,
+        message: str,
+        comment_to_existing: Optional[str] = None,
     ) -> Optional[Issue]:
         # TODO: Improve filtering
         issues = project.get_issue_list()
         title = f"[packit] {title}"
 
-        if any(title in issue.title for issue in issues):
-            return None
+        for issue in issues:
+            if title in issue.title:
+                if comment_to_existing:
+                    issue.comment(body=comment_to_existing)
```

Comment:
What happens with `message` here?

Reply:
It's not used. I wanted to differentiate the text used for issue description and issue comment.

I know it's not used yet but we can use a different text when creating the issue (e.g. add contacts) and avoid this to be mentioned in each comment.